specs/agents/trace_continuation.md

```diff
@@ -0,0 +1,15 @@
+## Trace Continuation
```

Comment:
nit: All other spec files use `-` rather than `_` in the name. I'd be included to use a hyphen. But I don't care strongly. :)

Reply:
Agreed - I looked at the other files and wanted to follow the convention. It seems I failed on that. Fixed now.